packages/api-client/src/giphy/GiphyAPI.ts

```diff
@@ -32,6 +32,21 @@ export class GiphyAPI {
     SEARCH: 'search',
   };
 
+  /**
+   * Get a GIF by ID
+   * @param ids one or multiple image ID(s)
```

Comment:
Are these image IDs or Giphy IDs? Where do I get such ID?

Reply:
By nature of the Giphy API this is a giphy ID. You can get such an ID by searching for a gif with the other two endpoints.